style.css

```diff
@@ -0,0 +1,180 @@
+/* stylelint-disable function-calc-no-unspaced-operator */
+/* stylelint-disable no-missing-end-of-source-newline */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable color-hex-case */
+/* stylelint-disable length-zero-no-unit */
+/* stylelint-disable indentation */
+
+body {
+  background-color: #1C1A19;
+  }
+
+@media (max-width: 375px) {
+.container {
+position: absolute;
+width: 375px;
+height: 92px;
+left: 0px;
+top: 0px;
+background: #3C3A39;
+}
+
+  .time {
+  position: absolute;
+  width: 54px;
+  height: 21px;
+  left: 21px;
+  top: 13px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  letter-spacing: -0.28px;
+
+  /* Color/4/100% */
+
+  color: #FFFFFF;
+  }
+
+  .cell {
+  position: absolute;
+  width: 17px;
+  height: 10.67px;
+  right: 64.33px;
+  top: 17.67px;
+  }
+
+  .wifi {
+  position: absolute;
+  width: 15.27px;
+  height: 10.97px;
+  right: 44.03px;
+  top: 17.33px;
+  }
+
+  .batt {
+    position: absolute;
+    width: 24.33px;
+    height: 11.33px;
+    left: 336px;
+    top: 17.33px;
+  }
+
+  .logo {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: absolute;
+    width: 94px;
+    height: 44px;
+    left: 12px;
+    top: calc(50% - 44px/2);
+  }
```

Comment:
- [ ] you are currently not making use of the appropriate fonts family for the logo as prescribed from the figma template, please endeavor to add the css property for roboto fonts before your next pr :+1: